packages/aws-cdk/bin/cdk.ts

```diff
@@ -71,7 +71,7 @@ async function parseCommandLineArguments() {
     .command('init [TEMPLATE]', 'Create a new, empty CDK project from a template. Invoked without TEMPLATE, the app template will be used.', yargs => yargs
       .option('language', { type: 'string', alias: 'l', desc: 'The language to be used for the new project (default can be configured in ~/.cdk.json)', choices: initTemplateLanuages })
       .option('list', { type: 'boolean', desc: 'List the available templates' })
-      .option('package-manager', { type: 'string', default: 'npm', desc: 'The Node.js package manager to use, e.g. "npm", "yarn", "pnpm". To be used with --language=typescript|javascript'}))
+      .option('generate-only', { type: 'boolean', default: false, desc: 'If true, the dependencies will not be automatically installed'}))
```

Comment:
Change `desc` to `only generates project files, without executing additional operations such as setting up a git repo, installing dependencies or compiling the project`.

Reply:
Done